setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import find_packages, setup
```

Comment:
please review the files you add to your project. review the PR before submitting it. if there is some project template - review it, analyze the files it has and why are they needed, analyze if they are needed in your case. and, if some template is used, make sure that it is correctly adapted for your project.

also, so far it was not required to package the project. what are the benefits of project packing?

Reply:
fixed